pyspecdata/plot_funcs/pcolormesh.py

```diff
@@ -19,12 +19,12 @@ def pcolormesh(
 
     Parameters
     ==========
-    fig: matplotlib figure object
+    fig : matplotlib figure object
     shading: str (default 'nearest')
         the type of shading to pass to matplotlib pcolormesh
-    ax1: matplotlib axes object
+    ax1 : matplotlib axes object
         where do you want the left plot to go?
-    ax2: matplotlib axes object
+    ax2 : matplotlib axes object
```

Comment:
please note that my format for numpy docstrings wasn't quite to spec, so I'm fixing it moving forward (that's what I'm doing here)